tests/manager/openAccount/managerCanOpenAccount.spec.js

```diff
@@ -28,4 +51,59 @@ Test:
 Tips:
  1. Do not rely on the customer row id for the step 13. Use the ".last()" locator to get the last row.
 */
-});
\ No newline at end of file
+const bankManagerMainPage = new BankManagerMainPage(page); 
+const openAccountPage = new OpenAccountPage(page); 
+const customersListPage= new CustomersListPage (page);
+
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickOpenAccountButton();
+await openAccountPage.waitForOpened();
+await openAccountPage.selectCustomer(customerName);
+await openAccountPage.selectCurrency('Dollar');
+await openAccountPage.clickProcessButton();
+await openAccountPage.reload();
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickCustomersButton();
+await customersListPage.waitForOpened();
+await customersListPage.assertLastRowAccountNumberContainsText('');
+
+});
+
+test('Assert manager can open pound account', async ({ page }) => {
+  const bankManagerMainPage = new BankManagerMainPage(page); 
+const openAccountPage = new OpenAccountPage(page); 
+const customersListPage= new CustomersListPage (page);
+
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickOpenAccountButton();
+await openAccountPage.waitForOpened();
+await openAccountPage.selectCustomer(customerName);
+await openAccountPage.selectCurrency('Pound');
+await openAccountPage.clickProcessButton();
+await openAccountPage.reload();
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickCustomersButton();
+await customersListPage.waitForOpened();
+await customersListPage.assertLastRowAccountNumberContainsText('');
+
+});
+
+test('Assert manager can open rupee account', async ({ page }) => {
+const bankManagerMainPage = new BankManagerMainPage(page); 
+const openAccountPage = new OpenAccountPage(page); 
+const customersListPage= new CustomersListPage (page);
+
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickOpenAccountButton();
+await openAccountPage.waitForOpened();
+await openAccountPage.selectCustomer(customerName);
+await openAccountPage.selectCurrency('Rupee');
+await openAccountPage.clickProcessButton();
+await openAccountPage.reload();
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickCustomersButton();
+await customersListPage.waitForOpened();
+await customersListPage.assertLastRowAccountNumberContainsText('');
```

Comment:
The assertions `assertLastRowAccountNumberContainsText('')` are checking if the account number is empty, which contradicts the test step that expects the account number to be not empty after opening an account. Consider changing the assertion to check that the account number is not empty.